src/pages/about/index.tsx

```diff
@@ -60,7 +59,7 @@ export async function getStaticProps(context: GetStaticPropsContext) {
 
   await cmsClient.pageQuery({
     nextContext: context,
-    query: LANDING_PAGE_QUERY,
+    query: ACCESSIBILITY_PAGE_QUERY,
```

Comment:
Is this query named in an odd way? Considering that this is the about page?

Reply:
Renamed now. Seems that I can't win with these copy pasted things on this PR 😄